app/handlers/signup_start.rb

```diff
@@ -34,6 +34,7 @@ def handle
 
     # Create a new one
     new_signup_state = SignupState.email_address.create(
+      is_partial_info_allowed: false,
```

Comment:
Don't allow partial info (incomplete contact info) once a user actually fills out the signup start form